### ISSUE TITLE
cascade_invocations does not fire inside switch statements

### ISSUE THREAD

**Issue by natebosch** [dart-lang/linter#5108]
The `cascade_invocations` lint will ignore statements that can use a cascade if they are inside a `switch`

```dart
void main() {
  var l = [];
  l.clear(); // Correctly flagged by lint
  switch (null) {
    default:
      var l = [];
      l.clear(); // Missed by lint
  }
}
```
